manta-pay/abstract-protocol.tex

```diff
@@ -190,133 +176,169 @@ \subsection{Abstract Cryptographic Schemes}
+The \Transfer{} protocol is the fundamental abstraction in \MantaPay{} and facilitiates the valid transfer of \zkAsset{s} among participants while preserving their anonymity. The \Transfer{} is made up of special cryptographic constructions called \Sender{s} and \Receiver{s} which represent the private input and the private output of a transaction. To perform a \Transfer{}, a protocol participant gathers the \SpendingKey{s} they own, selects a subset of the \UTXO{s} they have still not spent (with a fixed \AssetId{}), collects \ReceivingKey{s} from other participants for the outputs, assigning each key a subset of the input \zkAsset{s}, and then builds a \Transfer{} object representing the transfer they want to build. From this \Transfer{} object, they construct a \TransferPost{} which they then send to the \Ledger{} to be validated and stored, representing a completed state transition in the \Ledger{}. The transformation from \Transfer{} to \TransferPost{} involves keeping the parts of the \Transfer{} that \emph{must} be known to the \Ledger{} and for the parts that \emph{must} not be known, substituting them for a \emph{zero-knowledge proof} representing the validity of the secret information known to the participant, and the \Transfer{} as a whole.
 
 We begin by defining the cryptographic primitives involved in the \Transfer{} protocol:
 
 \begin{definition}[Transfer Configuration]
     A \TransferConfiguration{} is a collection of implementations of the following abstract cryptographic primitives:
     \begin{itemize}
-        \item \textbf{Hybrid Public Key Encryption}: $\HPKE$
-        \item \textbf{UTXO Commitment Scheme}: $\COM^{\UTXO}$
-        \item \textbf{Void Number Commitment Scheme}: $\COM^{\VN}$
-        \item \textbf{Dynamic Cryptographic Accumulator}: $\DCA$
+        \item \textbf{Key Schedule}: $\KeySchedule$
+        \item \textbf{Incoming Authenticated Encryption Scheme}: $\AUTHin$
+        \item \textbf{Outgoing Authenticated Encryption Scheme}: $\AUTHout$
```

Comment:
Add some connection to what has been explained before, i.e., to Encrypted Notes.